fe/fe-core/src/main/java/com/starrocks/sql/analyzer/FunctionAnalyzer.java

```diff
@@ -153,6 +153,12 @@ private static void analyzeBuiltinAggFunction(FunctionCallExpr functionCallExpr)
             }
         }
 
+        if (fnName.getFunction().equalsIgnoreCase(FunctionSet.ARRAY_FILTER)) {
```

Comment:
please change all `equalsIgnoreCase` to `equals`  in analyzeBuiltinAggFunction method. thanks.

Reply:
solved